plugins/wfp_auth/tests/test_auth.py

```diff
@@ -0,0 +1,134 @@
+from unittest.mock import patch
+
+from allauth.socialaccount.models import SocialAccount
+
+from iaso import models as m
+from iaso.test import APITestCase
+from plugins.wfp_auth.views import ExtraData
+
+
+class WFPAuthTestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.data_source = m.DataSource.objects.create(name="Data source")
+        cls.version = m.SourceVersion.objects.create(number=1, data_source=cls.data_source)
+        cls.account = m.Account.objects.create(name="Account", default_version=cls.version)
+        cls.project = m.Project.objects.create(name="Project", account=cls.account, app_id="test_app_id")
```

Comment:
Nitpick : you could use `cls.create_account_datasource_version_project` (but this might break some tests if other values are hardcoded)
 

```suggestion
        cls.account, cls.data_source, cls.version, cls.project = cls.create_account_datasource_version_project(
            source_name="Data source", account_name="Account", project_name="Project")
```